### DISCUSSION TITLE
I need help with New-HTMLTable and  New-TableRowGrouping

### DISCUSSION THREAD

**Discussion by gglgiggles** [EvotecIT/PSWriteHTML#266] [Q&A] [answered]
I have the following code that almost looks like I want.
```
New-HTML -TitleText $Title -Online -FilePath $PSScriptRoot\qualifiers.htm -Temporary {
New-HTMLSection -HeaderText 'Source fields For Qualified Converts' {
    New-HTMLTable -DataTable $qualified -HideFooter -PagingStyle full_numbers  -DisableOrdering -DisableAutoWidthOptimization {
        New-HTMLTableColumnOption -ColumnIndex 0 -Hidden $true
        New-HTMLTableColumnOption -ColumnIndex 1 -Width 20
        New-HTMLTableColumnOption -ColumnIndex 2 -Width 20
        New-HTMLTableColumnOption -ColumnIndex 3 -Width 20
        New-HTMLTableColumnOption -ColumnIndex 4 -Width 100
        New-HTMLTableColumnOption -ColumnIndex 5 -Width 10
        New-TableRowGrouping -Name 'client' -Color black -BackgroundColor LightBlue
    }
}
} -ShowHTML
```
Issue 1: When run, I get the groups but would like them all to be collapsed

![image](https://user-images.githubusercontent.com/79213776/130124322-bb05868c-9167-4055-a5d9-1c8c2997e3e3.png)

Issue 2: ColumnIndex 5 can be lengthy and I would like to force this off of the main screen and into something that can be clicked to be shown and have it wrapped as above.  The only way, I was able to do that was to EnableScroller, but then the data is not wrapped.

![image](https://user-images.githubusercontent.com/79213776/130124423-bcfbec14-972d-4edb-acbd-db846a22ad5e.png)

**Comment by PrzemyslawKlys** [comment, marked as answer]
Answered in https://github.com/EvotecIT/PSWriteHTML/issues/269
